README.md

```diff
@@ -164,36 +166,30 @@ The splunk_hec module allows the posting of PE orchestrator and activity service
 
```

Comment:
I know the line above this one is not a part of this PR, but it has trailing white space and it would be great if we could fix that nit while we're here.